CPP/Applications/Stack/Calculator/Calculator.cpp

```diff
@@ -0,0 +1,183 @@
+#include <stack> 
+#include <algorithm>
+#include <string>
+#include "Calculator.h"
+
+bool EvaluateInfix(string _Exression[] ,uint32_t Size , float *pResult )
+{
+    string _PostExression[Size];
+    uint32_t NewSize; 
+    bool Expression_Flag = true ; 
+    if( GetPostFix(_Exression ,  _PostExression , Size , &NewSize) )
+    {
+        (*pResult) = RevesePolishNotation(_PostExression , NewSize);
+    }
+    else
+    {
+        //Wrong Expression ;
+        Expression_Flag = false ;
+    }
+    return Expression_Flag ;
+}
+bool GetPostFix(string _Exression[]  , string _PostExression[]  , uint32_t Size , uint32_t *NewSize  ) 
```

Comment:
in cpp we prefer to avoid using pointers so you have to us passing by reference 
`bool GetPostFix(string _Exression[] , string _PostExression[] , uint32_t Size , uint32_t &NewSize )`
and when you call it 
`GetPostFix(..,..,..,size);`